src/docs/command-line.dox

```diff
@@ -202,6 +202,12 @@ the \c -f option re-directs the output to the specified file.
 Dump in JSON (<a href="http://www.json.org">JavaScript Object Notation</a>)
 format.
 
+@par \c -k
+The specific key to look for.
```

Comment:
In what format?

Reply:
Updated in 7dee5745d12577ff31dbd7d61d7616568a8f17db